.pre-commit-config.yaml

```diff
@@ -6,8 +6,8 @@ repos:
     - id: mixed-line-ending
       args: ['--fix=lf']
       description: Forces to replace line ending by the UNIX 'lf' character.
--   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v2.0.2
+-   repo: https://github.com/hhatto/autopep8
```

Comment:
I do not like this, either put description here or into the issue

Reply:
In the morning, I had a problem with pre-commit. When I opened the link https://github.com/pre-commit/mirrors-autopep8, I found information about an unsupported repository and the deprecation of the new one. I also added this information to the issue